src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -68,8 +65,6 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null): void
     }
 
     /**
-     * {@inheritdoc}
-     *
```

Comment:
As we override @return, we need to keep it, right?

Reply:
inheritane is implicit, so it can be removed

> will only be inherited if the redefined method's DocBlock does not have the element that is to be inherited (https://docs.phpdoc.org/guide/guides/inheritance.html)

moreover, i believe the correct return type is already added upstream. So could be removed as well, but im not sure tooling can detect it.